### PR TITLE
Add new lighthouse metrics

### PR DESCRIPTION
**Pull request by akarasavov**
Describe two new public metrics of lighthouse

@yirensum additional context - https://neo4j.atlassian.net/wiki/spaces/Cluster/pages/511606802/Lighthouse+GA+plan

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1692.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1692.surge.sh/operations-manual/5/monitoring/metrics/reference)

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 1]
```diff
@@ -437,6 +437,17 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
 |===
 
+[[discovery-metrics]]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:440]:
```suggestion
[[discovery-metrics-v2]]
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:440]: This will be a new metric class?

Comment (NataliaIvakina) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:440]: our test shows that we have to have the metric class - Lighthouse metrics. I suppose we should rename 'Discover metrics v2' into 'Lighthouse metrics'

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1692.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1692.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1692.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1692.surge.sh/operations-manual/5/monitoring/metrics/reference)

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 2]
```diff
@@ -423,9 +423,9 @@ label:deprecated[Deprecated in 5.15]
 |===
 
 [[discovery-metrics]]
-=== Discovery metrics
+=== Discovery metrics v1
 
-.Discovery metrics
+.Discovery metrics v1
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:428]: If you want to keep it, you need to change it in the code as well.

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 3]
```diff
@@ -437,6 +437,17 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
 |===
 
+[[lighthouse-metrics]]
+=== Discovery metrics v2
+
+.Discovery metrics v2
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:443]:
```suggestion
.Lighthouse metrics
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc (hunk 2 above)
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:428]:
```suggestion
.Discovery metrics
```

**Review by renetapopova — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 4]
```diff
@@ -437,6 +437,17 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
 |===
 
+[[lighthouse-metrics]]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:440]:
```suggestion
[role=label--new-5.22]
[[lighthouse-metrics]]
```

**Comment by akarasavov** [comment]
I have changed the documentation tag from two types of metrics - https://github.com/neo-technology/neo4j/pull/26065. 
There is no more lighthouse metrics tag. Now we have Discovery Service V1 and Discovery Service V2.
@renetapopova

File: modules/ROOT/pages/monitoring/metrics/reference.adoc (hunk 2 above)
Comment (akarasavov) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:428]: https://github.com/neo4j/docs-operations/pull/1692#issuecomment-2196629424

File: modules/ROOT/pages/monitoring/metrics/reference.adoc (hunk 3 above)
Comment (akarasavov) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:443]: > The title should reflect the code `@Documented(".Lighthouse metrics")`.

https://github.com/neo4j/docs-operations/pull/1692#issuecomment-2196629424

>What about the rest of the metrics? Will we document them as well for 5.22 or just these two?

Discovery service V2(Lighthouse) has only two public metrics. All other metrics that you see in the file are internal. We don't document internal metrics.

File: modules/ROOT/pages/monitoring/metrics/reference.adoc (hunk 4 above)
Comment (akarasavov) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:441]: done

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1692.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1692.surge.sh/operations-manual/5/monitoring/metrics/reference)

File: modules/ROOT/pages/monitoring/metrics/reference.adoc (hunk 2 above)
Comment (akarasavov) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:428]: Done here -> https://github.com/neo-technology/neo4j/pull/26065

Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:428]:
```suggestion
.Discovery service V1
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 5]
```diff
@@ -437,6 +437,18 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
 |===
 
+[role=label--new-5.22]
+[[lighthouse-metrics]]
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:441]:
```suggestion
[[discovery-service-V2]]
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1692.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1692.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1692.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1692.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Review by renetapopova — APPROVED** [review]
Thank you, @akarasavov!

**Pull request merged**